systemd/docker-lvm-plugin.service

```diff
@@ -5,8 +5,8 @@ Before=docker.service
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/docker-lvm-volumegroup
-ExecStart=/usr/bin/docker-lvm-plugin
+EnvironmentFile=-/etc/sysconfig/docker-lvm-plugin
```

Comment:
I know, this was already incorrect. Can you fix it ?
I think I already had a discussion with @rhatdan on this, and we finalized on `/etc/docker`.